codebuild_specs/sh-files/aws-configure-credentials.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
Since the Amplify CLI won't accept session credentials directly, the credentials need to be added to ~/.aws/credentials